sdk/communication/communication-sms/test/public/suites/smsClient.send.ts

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+import { SmsSendRequest, SmsSendOptions } from "../../../src";
+import { env } from "@azure/test-utils-recorder";
+import { Context } from "mocha";
+import { assertIsFailureResult, assertIsSuccessResult } from "../utils/assertHelpers";
+
+export default function suites(): void {
```

Comment:
Correct me if I'm wrong, these are test cases. A suite is a bunch of test cases wrapped in `describe`. So I think a more appropriate name would be `testCases`.

Reply:
good catch! I'll change this

Reply:
hmm, actually, we export this so that we can wrap it in a describe. that said, I do think the name could be different